src/CMakeLists.txt

```diff
@@ -453,9 +453,9 @@ install(TARGETS ${installable_targets}
 unset(installable_targets)
 
 if(INSTALL_MAN)
-  # TODO: these stubs are no longer needed. man pages should be generated at install time.
-  install(DIRECTORY ../doc/man/
+  install(CODE "execute_process(COMMAND ${CMAKE_COMMAND} -E make_directory ${PROJECT_BINARY_DIR}/doc/man)")
+  install(CODE "execute_process(COMMAND ${CMAKE_COMMAND} -E env BUILDDIR=${PROJECT_BINARY_DIR} MANDIR=${PROJECT_BINARY_DIR}/doc/man ${PROJECT_SOURCE_DIR}/contrib/devtools/gen-manpages.py --skip-missing-binaries)")
```

Comment:
> --skip-missing-binaries

I don't think we should be hardcoding a flag to ignore/skip errors into a script that you're making a requirement for release builds.